### ISSUE TITLE
Fix Chat Component UI

### ISSUE THREAD

**Issue by Saachi26** [MadhavKrishanGoswami/InkCode-Fusion#20]
The chat component's UI requires improvements to ensure better readability and user experience. 
Specifically:

- Make the text boxes proportionate to the text.
- The text boxes should resize dynamically according to the text content.

![Screenshot from 2024-10-06 22-30-22](https://github.com/user-attachments/assets/e9b57cda-7cfe-4804-baea-42b19cfa1c5f)

**Tasks**

- Dynamically resize text boxes based on the text length.
- Remove the extra scroll bar from the chat component.

**Comment by nunniii** [comment]
can I work in this ?

**Comment by MadhavKrishanGoswami** [comment]
go ahead @nunniii!

**Comment by nunniii** [comment]
Good morning, @MadhavKrishanGoswami! 🌅 , please consider testing the chat changes made in this [fork](https://github.com/nunniii/InkCode-Fusion) so I can create a PR.

- I was unable to observe the divs that contain the message content of "other" (classes .other) as I had difficulty navigating the project.
- Please also note that the file preventHorizontalScroll.js was added, which is a tool to address the issue of a horizontal scroll bar being rendered whenever the user's window is resized or when the stylesheets are modified. I’m not sure of the best way to resolve this issue, but this file has been effective in preventing the scroll.
- There are issues with the floating logic applied in the stylesheet: I believe it’s worth mentioning, even though it seems to be just a detail, I couldn’t find a solution for this part:

![image](https://github.com/user-attachments/assets/89ef4d71-66ca-4ad9-bf9a-44268de7c99b)

It remains hidden:
![image](https://github.com/user-attachments/assets/d85c9243-97fb-4edc-92a4-1cc6d142f153)


**In summary, here’s a preview:**

![image](https://github.com/user-attachments/assets/d0a276e2-9ec8-492f-a8e2-7c4895668252)

**Comment by Saachi26** [comment]
Thanks for your work on this issue! To properly review and test what you've worked on, could you please create a pull request? Once the PR is up, we can go through the changes and ensure everything is functioning as expected.

**Comment by nunniii** [comment]
alright! 💻

**Comment by Jasmeetkaur-13** [comment]
Could you please assign me to the task of fixing the Chat Component UI. I'm ready to work on improving the text box proportions and making them dynamically resize according to the text content

**Comment by MadhavKrishanGoswami** [comment]
Absolutely, @Jasmeetkaur-13  You’re assigned. Let me know if you need anything. 🚀

**Comment by Jasmeetkaur-13** [comment]
Thank you for assigning me this task! I'll make sure to focus on improving
the Chat Component UI, especially the dynamic resizing of the text box. If
I come across any challenges or need assistance, I'll definitely reach out.
Looking forward to delivering the best result! 😊

**Comment by Jasmeetkaur-13** [comment]
Thank you so muchh

**Comment by anchalchaturvedi08** [comment]
Can you assign me this task too?

**Comment by MadhavKrishanGoswami** [comment]
Sure, if you'd like, you can collaborate with @Jasmeetkaur-13 , as she's already working on this task.
